### PR TITLE
Add debouce to example

### PR DESCRIPTION
**Pull request by arturoromeroslc**
**What**:
Adding debounce to Axios example

**Why**:
It was commented to add it, not necessary.

**How**:
N/A

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: stories/examples/axios.js [hunk 1]
```diff
@@ -36,23 +36,23 @@ class AxiosAutocomplete extends Component {
...
-                      .then(response => {
-                        const items = response.data.items.map(
-                          item => `${item.name} (id:${item.id.toString()})`,
-                        ) // Added ID to make it unique
-                        this.setState({items})
-                      })
-                      .catch(error => {
-                        console.log(error)
-                      })
+                    setTimeout(() => {
```
Comment (kentcdodds) [inline stories/examples/axios.js]: Sorry, but `setTimeout` is not an efficient denounce method. You could copy/paste the one we have in the utils file.

File: src/__tests__/downshift.props.js [hunk 2]
```diff
@@ -97,10 +97,7 @@ test('onInputValueChange called when changes contain inputValue', () => {
   })
   selectItem('foo')
   expect(handleInputValueChange).toHaveBeenCalledTimes(1)
-  expect(handleInputValueChange).toHaveBeenCalledWith(
-    'foo',
-    expect.any(Object),
-  )
+  expect(handleInputValueChange).toHaveBeenCalledWith('foo', expect.any(Object))
```
Comment (arturoromeroslc) [inline src/__tests__/downshift.props.js]: Not sure why this formatting happened.

**Comment by arturoromeroslc** [comment]
Done, and thank you for looking at it.  Always like contributing to your repos they are very approachable to new devs 👍.

File: src/__tests__/downshift.props.js (hunk 2 above)
Comment (kentcdodds) [reply src/__tests__/downshift.props.js]: No worries

**Pull request merged**